Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -385,7 +385,7 @@ void MediaSource::completeSeek()
     MediaTimePromise::AutoRejectProducer producer(PlatformMediaError::SourceRemoved);
     Ref promise = producer.promise();
 
-    scriptExecutionContext()->enqueueTaskWhenSettled(SourceBuffer::ComputeSeekPromise::all(WTF::map(*m_activeSourceBuffers, [&](auto&& sourceBuffer) {
+    scriptExecutionContext()->enqueueTaskWhenSettled(SourceBuffer::ComputeSeekPromise::all(WTF::map(*activeSourceBuffers(), [&](auto&& sourceBuffer) {
```

Comment:
Can we make activeSourceBuffers() pass a reference, or just use m_activeSourceBuffers instead.
Ditto elsewhere.

Reply:
`activeSourceBuffers()` and `sourceBuffers()` are JS interfaces. It requires to return a pointer, a Ref or RefPtr.

Returning a Ref or RefPtr would error in expression such as `for (auto& sourceBuffer : sourceBuffers())` as there's no template deduction happening there.

I thought it looked nicer to use `*sourceBuffers()` considering it's never null